rfcs/0070-introducing-decl-buffer.md

```diff
@@ -146,9 +146,36 @@ Specifically, the updated flow of buffer flattening using `DeclBuffer` will be:
 with flattened indices.
 
 ## TVM script updates
+* New statement `T.decl_buffer` will be introduced. It has the same interface as `T.buffer_decl`.
+```python
+def decl_buffer(
+    shape: Sequence[Union[PrimExpr, int]],
+    dtype: str = "float32",
+    data: Var = None,
```

Comment:
just one note--usually this is first if positional args are used (e.g. in the example above), is the order of these kwargs correct?

Reply:
I'm keeping the order the same as the existing `T.buffer_decl`

Reply:
ok, but in the example above (line 135) you have:
```
A_flattened = T.decl_buffer(A.data, (256,), "float32")
```

this says to me that:
- `shape` == `A.data`
- `dtype` == `(256,)`
- `data` == `"float32"`

could you keep the RFC consistent?

Reply:
Thank you for pointing out, I've updated the RFC